packages/domain-picker/src/domain-picker/domain-i-own-item.tsx

```diff
@@ -0,0 +1,63 @@
+/**
+ * External dependencies
+ */
+import * as React from 'react';
+import { __ } from '@wordpress/i18n';
+import { Icon, chevronRight } from '@wordpress/icons';
+import { createInterpolateElement } from '@wordpress/element';
+import { useLocalizeUrl } from '@automattic/i18n-utils';
+
+/**
+ * Internal dependencies
+ */
+
+interface Props {
+	siteSlug: string;
+	source: string;
+}
+
+const DomainIOwn: React.FunctionComponent< Props > = ( { siteSlug, source } ) => {
+	const localizeUrl = useLocalizeUrl();
+
+	return (
+		<label className="domain-picker__suggestion-item contains-link">
+			<div className="domain-picker__suggestion-item-name">
+				<span className="domain-picker__domain-wrapper with-margin">
+					{ createInterpolateElement(
+						__( '<strong>Already own a domain?</strong>', __i18n_text_domain__ ),
+						{
+							strong: <strong />,
+						}
+					) }
```

Comment:
same as https://github.com/Automattic/wp-calypso/pull/48119/files#r547204888

And as a side note, do we really want to use `strong` at all? Recently we've been using `font-weight: 500` as an exception for these bolded texts. Eg: https://github.com/Automattic/wp-calypso/blob/trunk/packages/domain-picker/src/domain-picker/style.scss#L270